src/app/Shared/Model/Client_Isolate.model.ts

```diff
@@ -2,8 +2,37 @@ export type DbCollection = {
     microorganism: string;
     samplingContext: string;
     matrix: string;
+    federalState: string;
+    samplingStage: string;
+    origin: string;
+    category: string;
+    productionType: string;
 }[];
 
-export type DbKey = "microorganism" | "samplingContext" | "matrix";
+export type DbKey =
+    | "microorganism"
+    | "samplingContext"
+    | "matrix"
+    | "federalState"
+    | "samplingStage"
+    | "origin"
+    | "category"
+    | "productionType"
 
-export const DbKeyCollection: DbKey[] = ["microorganism", "samplingContext", "matrix"];
+export const DbKeyCollection: DbKey[] = [
+    "microorganism",
+    "samplingContext",
+    "matrix",
+    "federalState",
+    "samplingStage",
+    "origin",
+    "category",
+    "productionType",
+];
+
+export interface ClientIsolateCountedDTO {

```

Comment:
Why is this here? It's a DTO and should be where the API models are.

Reply:
It's done in the commit "Add sampling year filter".